### PR TITLE
Fix persmissions for Wayland and secret storage

### LINKED ISSUE #69: Enable wayland support
Tutanota currently uses Electron 16, which has pretty solid Wayland support. All I needed to do was to add 
```
 --enable-features=UseOzonePlatform --ozone-platform=wayland
```
to the command. Maybe we could include a [wrapper script](https://github.com/flathub/org.signal.Signal/blob/master/signal-desktop.sh#L5-L8) like Signal did?

### LINKED ISSUE #77: Log in credentials required at each launch
Since today's update, every time I close the app when I launch it again it requires me to fill in the email address and password (even if I previously checked the "store password" box). There's a message below the log in button that states "The system keychain has been invalidated. Deleting stored credentials".

Tutanota v3.91.10 running on Debian 11 stable (flatpak backport 1.12.4-1~bpo11+1).

Thanks for your attention!

### PR DESCRIPTION
**Pull request by charlag**
close #77
close #69
see #135

### CODE REVIEW

File: manifest-template.js [hunk 1]
```diff
@@ -15,8 +15,10 @@ const manifest = {
 	"finish-args": [
 		"--share=ipc",
 		"--device=dri",
-		"--socket=x11",
+		"--socket=wayland",
+		"--socket=fallback-x11",
 		"--socket=pulseaudio",
+		"--socket=system-bus",
```
Comment (bbhtt) [inline manifest-template.js:21]: This isn't allowed at all. You never should use this.

Comment (hfiguiere) [reply manifest-template.js:21]: you need ti use `--log-session-bus` to figure out the name you are missing.

Comment (hfiguiere) [reply manifest-template.js:21]: just to clarify this will break the build the linter WILL error out.

**Comment by bbhtt** [comment]
@charlag You have org.kde.statusnotifierwatcher listed twice in finish args in the main manifest.

https://github.com/flathub/com.tutanota.Tutanota/blob/3e0c80e5bb43d52351cab30f650a35ef6018fe9a/com.tutanota.Tutanota.json#L10-L34

**Comment by charlag** [comment]
@bbhtt thanks!

**Comment by bbhtt** [comment]
Also you can just build with -Dgcrypt=false instead of patching, that is only used if libsecret is built with gcrypt.

**Pull request merged**